apps/cloudfront-functions/package.json

```diff
@@ -7,7 +7,7 @@
     "compile": "npm run clean && tsc",
     "lint": "eslint src",
     "pretest": "npm run compile",
-    "test": "vitest"
+    "test": "npm run compile && vitest"
```

Comment:
This is not necessary: there is the `pretest` script that does what you want.

Reference: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts